docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Can we somehow use "ott.tools.gaussian_mixture.gaussian_mixture.GaussianMixture.from_random(...) to avoid some code duplication?
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>